docker/README.md

```diff
@@ -68,14 +88,34 @@ $ git clone git@github.com:openfoodfoundation/openfoodnetwork.git
 ```command
 $ cd openfoodnetwork
 ```
-* Download the Docker images, build the Docker containers, seed the database with sample data, AND log the screen output from these tasks:
+* You have two choices of images you can build. The default Ubuntu image is most similar to the production servers but is larger in size and will take longer to build. Alternatively, you can use the Alpine image which is smaller, faster to setup, and should be compatible with the Ubuntu. To download the Ubuntu Docker images, build the Docker containers, seed the database with sample data, AND log the screen output from these tasks:
 ```command
 $ docker/build.ps1
 ```
+* To use the alpine image to start with append the name of the Dockerfile:
+```sh
+$ docker/build.ps1 alpine.Dockerfile
+```
 * Run the Rails server and its required Docker containers:
 ```command
 $ docker/server.ps1
 ```
+* Alternatively, you can run the server in detached mode. This will return you to the terminal prompt without stopping the server. No logs will be displayed:
+```sh
+$ docker compose up -d
+```
+* To stop the server, run:
+```sh
+$ docker compose down
+```
+* To access the server logs, run:
+```sh
+$ docker compose logs -f
+```
+* To run tests or access the server directly, you can use bash:
+```sh
+$ docker compose exec web bash
+```
```

Comment:
These commands look much the same between Linux and Windows, we should consider combining the sections in the future.